### PR TITLE
Fix debug

### PR DESCRIPTION
**Pull request by TApplencourt**
`--debug` is working again,. Thanks, Philipe!

### CODE REVIEW

File: utils/babeltrace_thapi.in [hunk 1]
```diff
@@ -75,19 +75,37 @@ module BTComponentClassRefinement
   refine(BT2::BTGraph) do
     attr_accessor :plugins_path, :cli_v
 
+    def bt2_escape(str, escapes = '*?[.:')
+      str.chars.map do |c|
+        escapes.include?(c) ? '\\' + c : c
+      end.join
+    end
```
Comment (Kerilk) [inline utils/babeltrace_thapi.in:82]:
```suggestion
    def bt2_escape(str)
      str.gsub(/([*?\[.:\\])/,'\\\\\1')
    end
```
Ca me parait mieux.

Comment (Kerilk) [reply utils/babeltrace_thapi.in:82]: Il te manquait l'escape de '\' aussi.

Comment (Kerilk) [reply utils/babeltrace_thapi.in:82]: test:
```ruby
s = 'to*to?ta[ta.tu:tu*ti\ti'
puts s.gsub(/([*?\[.:\\])/,'\\\\\1')
```
donne:
```
to\*to\?ta\[ta\.tu\:tu\*ti\\ti
```

Comment (TApplencourt) [reply utils/babeltrace_thapi.in]: So many escape :D But fit on one line, so :)

**Review by Kerilk — APPROVED** [review]

**Pull request merged**